localflavor/br/forms.py

```diff
@@ -79,6 +71,8 @@ class BRCPFField(CharField):
 
     More information:
     http://en.wikipedia.org/wiki/Cadastro_de_Pessoas_F%C3%ADsicas
+
+    .. versionchanged:: 2.1
```

Comment:
A short note about what changed is useful. Something like "Use BRCPFValidator for validation." would work but feel free to write something yourself. The same applies to other `versionchanged` directives.

http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionchanged